featuretools/tests/demo_tests/test_demo_data.py

```diff
@@ -24,3 +25,12 @@ def test_load_retail_diff():
     assert es_second['order_products'].df.shape[0] == nrows_second
     os.remove(make_retail_pathname(nrows))
     os.remove(make_retail_pathname(nrows_second))
+
+
+def test_load_mock_customer_dfs():
+    es = load_mock_customer(return_entityset=True)
+    dfs(entityset=es,
+        target_entity="sessions",
+        agg_primitives=["mean", "sum", "mode"],
+        trans_primitives=["month", "hour"],
+        max_depth=2)
```

Comment:
A little weird that this test has no assertions. Maybe assert that the normalize successfully converted both parent and child values to matching types?  Or maybe the test should go in test_es:test_add_relationship_converts_types